### DISCUSSION TITLE
aiofiles

### DISCUSSION THREAD

**Discussion by lathkar** [fastapi/fastapi#5463] [Questions] [answered]
Is aiofiles module not needed anymore to serve the static files?

**Comment by ReznikovRoman** [comment, marked as answer]
@lathkar Hi! Yep, `aiofiles` dependency has been [removed this year](https://github.com/turettn/fastapi_offline/pull/24)

**Reply by tiangolo** [reply]
Correct! After moving to AnyIO it's no longer needed. 🚀
